src/Microsoft.ML/PredictionModel.cs

```diff
@@ -29,6 +29,33 @@ internal Runtime.EntryPoints.TransformModel PredictorModel
             get { return _predictorModel; }
         }
 
+        /// <summary>
+        /// Returns labels that correspond to indices of the score array in the case of 
+        /// multi-class classification problem.
+        /// </summary>
+        /// <param name="mapping">Label to score mapping</param>
+        /// <param name="scoreColumnName">Name of the score column</param>
+        /// <returns></returns>
+        public bool TryGetScoreLabelMapping(out string[] mapping, string scoreColumnName = "Score")
+        {
+            mapping = null;
+            IDataView idv = _predictorModel.Schema;
+            int colIndex = -1;
+            if (!idv.Schema.TryGetColumnIndex(scoreColumnName, out colIndex))
+                return false;
+
+            VBuffer<DvText> labels = default(VBuffer<DvText>);
```

Comment:
>default(VBuffer<DvText>); [](start = 37, length = 25)

Can be just `default` since you were explicit with the type declaration. (New in some version of C# 7.x.) Or maybe just have this be `var`.